src/uwtools/drivers/ungrib.py

```diff
@@ -41,18 +42,27 @@ def __init__(
 
     # Workflow tasks
 
-    @task
-    def gribfile(self):
-        """
-        A symlink to the input GRIB file.
-        """
-        path = self._rundir / "GRIBFILE.AAA"
-        yield self._taskname(str(path))
-        yield asset(path, path.is_symlink)
-        infile = Path(self._driver_config["gfs_file"])
-        yield file(path=infile)
-        path.parent.mkdir(parents=True, exist_ok=True)
-        path.symlink_to(infile)
+    @tasks
+    def gribfiles(self):
+        """
+        Symlinks to all the GRIB files.
+        """
+        yield self._taskname("gribfiles")
+        gfs_files = self._driver_config["gfs_files"]
+        offset = abs(gfs_files["offset"])
+        endhour = gfs_files["forecast_length"] + offset + 1
+        interval = gfs_files["interval_hours"]
+        cycle_hour = int((self._cycle - timedelta(hours=offset)).strftime("%H"))
+        suffix = "AAA"
```

Comment:
Your call, but here's a function that maps integers to 3-letter strings of of the kind in use here:
```
def ext(n):
    b, c = 26, lambda x: chr(65 + x)
    return "{:A>3}".format((ext(n // b) if n >= b else "") + c(n % b))[-3:]
```
If we enumerated the GRIB files, we could use the index to produce the extensions with somewhat less code.

Reply:
Simpler:
```
def ext(n):
    b = 26
    return "{:A>3}".format(("" if n < b else ext(n // b)) + chr(65 + n % b))[-3:]
```